src/NUnit.Xml.TestLogger.TestAdapter/NUnit.Xml.TestLogger.TestAdapter.csproj

```diff
@@ -26,7 +26,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.TestPlatform.ObjectModel" Version="15.0.0" />
+    <PackageReference Include="Microsoft.TestPlatform.ObjectModel" Version="15.9.2" />
```

Comment:
Do we need the version update for dependency on any newer APIs? Would `15.5.0` version have the APIs?

Downside of moving to `15.9.2` is the logger extension will not be usable by older versions of `dotnet test`.

Reply:
15.5.0 seems to work!